coverage/.resultset.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Please remove this file. It should be in the gitignore on master now too, so it should not continue to be added.